src/main/java/com/scalar/db/storage/jdbc/query/SimpleSelectQuery.java

```diff
@@ -100,15 +101,17 @@ private String orderBySqlString() {
     List<Scan.Ordering> orderingList = new ArrayList<>(orderings);
 
     Boolean reverse = null;
-    for (int i = 0; i < tableMetadata.getClusteringKeys().size(); i++) {
+    Iterator<String> iterator = tableMetadata.getClusteringKeyNames().iterator();
```

Comment:
Just a question.
Is there any particular reason for using `iterator` instead of `for`?

Reply:
I'm not sure why I did it... Maybe I was tired... I will fix it with `for`. Thanks!